template/chat.html

```diff
@@ -34,6 +34,9 @@
         }
       },
 
+      /*
+        writes a message in the chat box.
+      */
       sendMessage: function(user, message) {
         if (message != '')
```

Comment:
I would check the string is not undefined and not null to start. Then the trickery with the empty spaces, at least trimming at start and at end and checking if is it is different from the empty string. At least.